app/views/podcasts/_podcast.html.erb

```diff
@@ -24,6 +24,20 @@
         <h5 class="card-title fs-1 display-1"><%= @podcast.title %></h5>
         <h6 class="card-subtitle mb-2 text-body-secondary fs-6"><%= @podcast.created_at %></h6>
         <p class="card-text fs-5 display-2"><%= @podcast.description %></p>
+        <p class="card-text fs-5 display-2">User_id: <%= @podcast.user_id %></p>
+        <% if current_user && current_user.subscriptions.exists?(podcast: @podcast) %>
+          <%= button_to podcast_subscription_path(@podcast, 1),
+            method: :delete,
+            class: "btn btn-outline-success mb-2" do %>
+            Unsubscribe <i class="bi bi-hand-thumbs-up-fill"></i>
+          <% end %>
+        <% else %>
+          <%= button_to podcast_subscriptions_path(@podcast),
+            method: :post,
+            class: "btn btn-outline-success mb-2" do %>
+            Subscribe <i class="bi bi-hand-thumbs-up"></i>
+          <% end %>
```

Comment:
let's use icons instead of text.

if need subscribe - https://icons.getbootstrap.com/icons/check-square/
is subscribed https://icons.getbootstrap.com/icons/check-square-fill/